draft-ietf-httpbis-digest-headers.md

```diff
@@ -492,6 +492,13 @@ A sender MAY send a representation-data-digest using a digest-algorithm without
 knowing whether the recipient supports the digest-algorithm, or even
 knowing that the recipient will ignore it.
 
+Examples:
+
+~~~
+  Digest: sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
+  Digest: id-sha-512=WZDPaVn/7XgHaAy8pmojAkGWoRx2UFChF41A2svX+TaPm+AbwAgBWnrIiYllu7BNNyealdVLvRwE\nmTHWXvJwew==
+  Digest: sha-256=4REjxQ4yrqUVicfSKYNO/cF9zNj5ANbzgDZt3/h3Qxo=, id-sha-256=X48E9qOokqqrvdts8nOJRJN3OWDUoyWxBf7kbu9DBPE=
+~~~
 ...
```

Comment:
I'd drop the first line (sha-256=X48...) and keep the other two.
It is probably legal to have multiple Digest headers so this looks like it might be 1 HTTP message (lines from 3 separate messages). But having sha-256 twice with different values in 1 message wouldn't make sense.

Reply:
I can clarify those are three different HTTP messages.
Ok for leaving just two.